tests/tests/test-rpc/test-starknet-rpc.ts

```diff
@@ -1146,4 +1146,22 @@ describeDevMadara("Starknet RPC", (context) => {
       });
     });
   });
+
+  describe("Fix #551: Madara RPC doesn't handle 'pending' block id", async () => {
+    it("should support 'pending' block id", async function () {
```

Comment:
could you maybe put this then in the used rpc method?

```
describe("getNonce", async () => {
    it("shoule return same nonce for pending and latest", async () => {
        /// ...
    })
})

Reply:
For me this test is not really related to `getNonce` that why I put in another group.
But for sure I can move it in `getNonce` group if it makes more sense.